ModelBundle/Repository/RedirectionRepository.php

```diff
@@ -12,4 +12,21 @@
 class RedirectionRepository extends AbstractAggregateRepository implements RedirectionRepositoryInterface
 {
     use PaginationTrait;
+
+    /**
+     * @param string $nodeId
+     * @param string $language
+     *
+     * @return array
+     */
+    public function findByNode($nodeId, $language){
+        $qa = $this->createAggregationQuery();
+
+        $qa->match(array(
+            'nodeId' => $nodeId,
+            'locale' => $language,
```

Comment:
In fact $language come from language property in node collection. Bu it's stocked as locale in redirection. Don't no why. Wasn't here ^^. But I will rename in repository context.